testnet/launcher/docker.go

```diff
@@ -149,6 +151,55 @@ func (t *Testnet) Start() error {
 	}
 	fmt.Println("L2 Contracts were successfully deployed...")
 
+	faucetInst, err := faucet.NewDockerFaucet(
+		faucet.NewFaucetConfig(
+			faucet.WithFaucetPort(99),
+			faucet.WithTenNodePort(13010),
+			faucet.WithTenNodeHost("validator-host"),
+			faucet.WithFaucetPrivKey("0x8dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b"),
+			faucet.WithDockerImage("testnetobscuronet.azurecr.io/obscuronet/faucet:latest"),
+		),
+	)
+	if err != nil {
+		return fmt.Errorf("unable to instantiate faucet - %w", err)
+	}
+
+	if err = faucetInst.Start(); err != nil {
+		return fmt.Errorf("unable to start faucet - %w", err)
+	}
+
+	if err = faucetInst.IsReady(); err != nil {
+		return fmt.Errorf("unable to wait for faucet to be ready - %w", err)
+	}
+
+	fmt.Println("Faucet ready to be accessed at http://127.0.0.1:99/ ...")
```

Comment:
This port number should be read off the config or from constant probably, same with the gateway one below.

Reply:
The configs don't have external reading, but I agree this should be some form of constant. Changed 👍